src/sidebar/components/Annotation/test/AnnotationActionBar-test.js

```diff
@@ -52,7 +52,9 @@ describe('AnnotationActionBar', () => {
   };
 
   const getButton = (wrapper, iconName) => {
-    return wrapper.find('IconButton').filter({ icon: iconName });
+    return wrapper
+      .find('IconButton')
+      .filterWhere(n => n.find(iconName).exists());
```

Comment:
It could be simpler just to give these buttons `data-testid` props that can be matched with an attribute selector.